hbase-client/src/test/java/org/apache/hadoop/hbase/shaded/protobuf/TestProtobufUtil.java

```diff
@@ -623,4 +626,149 @@ public void testSlowLogParamsMutateRequest() {
+      .setRegion(givenRegion()).setGet(ProtobufUtil.toGet(new Get(Bytes.toBytes("foo")))).build();
+
+    String actual = ProtobufUtil.getShortTextFormat(getRequest);
+
+    assertNotNull(actual);
+    assertEquals("region= test, row=foo", actual);
+  }
+
+  @Test
+  public void testGetShortTextFormatMultiRequest() throws IOException {
+    ClientProtos.Action action = ClientProtos.Action.newBuilder()
+      .setGet(ProtobufUtil.toGet(new Get(Bytes.toBytes("foo")))).build();
+    ClientProtos.RegionAction regionAction =
+      ClientProtos.RegionAction.newBuilder().addAction(action).setRegion(givenRegion()).build();
+
+    ClientProtos.MultiRequest multiRequest =
+      ClientProtos.MultiRequest.newBuilder().addRegionAction(regionAction).build();
+
+    String actual = ProtobufUtil.getShortTextFormat(multiRequest);
+
+    assertNotNull(actual);
+    assertEquals("region= test, for 1 action(s) and 1st row key=foo", actual);
+  }
+
+  @Test
+  public void testGetShortTextFormatMutateRequest() throws IOException {
+    ClientProtos.MutateRequest mutateRequest = ClientProtos.MutateRequest.newBuilder()
+      .setMutation(
+        ProtobufUtil.toMutation(MutationType.INCREMENT, new Increment(Bytes.toBytes("foo"))))
+      .setRegion(givenRegion()).build();
+
+    String actual = ProtobufUtil.getShortTextFormat(mutateRequest);
+
+    assertNotNull(actual);
+    assertEquals("region= test, row=foo", actual);
+  }
+
+  @Test
+  public void testGetShortTextFormatCoprocessorServiceRequest() {
+    ClientProtos.CoprocessorServiceCall call = ClientProtos.CoprocessorServiceCall.newBuilder()
+      .setRow(UnsafeByteOperations.unsafeWrap(Bytes.toBytes("foo"))).setMethodName("awesomeMethod")
+      .setServiceName("awesomeService")
+      .setRequest(UnsafeByteOperations.unsafeWrap(Bytes.toBytes("foo-request"))).build();
+
+    ClientProtos.CoprocessorServiceRequest.Builder builder =
+      ClientProtos.CoprocessorServiceRequest.newBuilder();
+    builder.setRegion(givenRegion());
+    builder.setCall(call);
+    ClientProtos.CoprocessorServiceRequest coprocessorServiceRequest = builder.build();
+
+    String actual = ProtobufUtil.getShortTextFormat(coprocessorServiceRequest);
+
+    assertNotNull(actual);
+    assertEquals("coprocessorService= awesomeService:awesomeMethod", actual);
+  }
+
+  @Test
+  public void testGetShortTextFormatMoveRegionRequest() {
+    MasterProtos.MoveRegionRequest.Builder builder = MasterProtos.MoveRegionRequest.newBuilder();
+    builder.setRegion(givenRegion());
+    builder.setDestServerName(givenServerName());
+    MasterProtos.MoveRegionRequest moveRegionRequest = builder.build();
+
+    String actual = ProtobufUtil.getShortTextFormat(moveRegionRequest);
+
+    assertNotNull(actual);
+    assertEquals(
+      "region { type: REGION_NAME value: \"test\" } dest_server_name { host_name: \"a.b.com\" }",
+      actual);
+  }
+
+  private static HBaseProtos.ServerName givenServerName() {
```

Comment:
You could define class constants for these instead of these static methods, like TAG_STR is handled.